README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="/assets/stark-core/logo/stark_logo_black.png" alt="Stark : a powerful front-end framework" width="150" height="60">
```

Comment:
This path is wrong... it is pointing to an "assets" folder in the root, but the image is in this path: "packages/stark-core/assets/logo/stark_logo_black.png"